Card/Expiry.ts

```diff
@@ -10,4 +11,8 @@ export type Expiry = [Year, Month]
 export namespace Expiry {
 	export const type = isly.tuple<Expiry>(isly.number([...year]), isly.number([...month])) // Deconstructing to remove readonly.
 	export const is = type.is
+	export function toDateTime(expiry: Expiry): isoly.DateTime {
+		const month = expiry[1].toString().length == 2 ? `${expiry[1]}` : `0${expiry[1]}`
+		return isoly.DateTime.nextMonth(`20${expiry[0]}-${month}-01T00:00:01.000Z`)
```

Comment:
This is not as important as the one above but shouldn't it be the last second of the last day on the month? so if you put 
```
isoly.DateTime.previousDay(isoly.DateTime.nextMonth(`20${expiry[0]}-${month}-01T23:59:59.999Z`))
```

Reply:
Fixing this is optional.